mm2src/mm2_gui_storage/src/rpc_commands.rs

```diff
@@ -172,7 +172,8 @@ pub struct SetBalanceRequest {
 ///
 /// This RPC affects the storage **only**. It doesn't affect MarketMaker.
 pub async fn enable_account(ctx: MmArc, req: EnableAccountRequest) -> MmResult<SuccessResponse, AccountRpcError> {
-    let account_ctx = AccountContext::from_ctx(&ctx).map_to_mm(AccountRpcError::Internal)?;
+    // TODO db_id
+    let account_ctx = AccountContext::from_ctx(&ctx, None).map_to_mm(AccountRpcError::Internal)?;
```

Comment:
I guess there is no way for this function to figure out db_id, if I am not mistaken? If I am right, we should probably extend the function args with `db_id` arg.

Reply:
Yeah there're some other rpc methods like this too.. cc @shamardy

Reply:
This is part of GUI storage methods which was done by Sergey a long time ago and we later decided to not utilize it for HD wallet. GUI storage methods can be safely removed from the codebase, if we ever need it in the future, we can get it from commits history but my implementation approach for HD wallets will not need it as far as I can see.

Reply:
I guess just comment them out?